### ISSUE TITLE
"Return type hint can be used" -> Import class

### ISSUE THREAD

**Issue by donquixote** [kalessil/phpinspectionsea#703]
Currently the "Return type hint can be used" bulk fix inserts the FQCN.
Instead, it should import the class.

**Comment by funivan** [comment]
Can you provide the code sample?

**Comment by donquixote** [comment]
> Can you provide the code sample?

Before:
```php
interface I {
  /**
   * @return \Acme\Animal\Cow
   */
  public function foo();
```
Expected after applying the fix:
```php
use Acme\Animal\Cow;

interface I {
  /**
   * @return \Acme\Animal\Cow
   */
  public function foo(): Cow;
```
Actual:
```php
interface I {
  /**
   * @return \Acme\Animal\Cow
   */
  public function foo(): \Acme\Animal\Cow;
```

**Comment by kalessil** [comment]
Right, return type hint inspection doesn't support auto-import yet (but ::class usage does). We have multiple tickets duplicating this PR, hence closing this one.

**Comment by rentalhost** [comment]
It is related to #320 and #255?
